### PR TITLE
Added markdownlint workflow

### PR DESCRIPTION
**Pull request by framtale**

### CODE REVIEW

File: .github/rules/markdownlint.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "tables": true,
+        "headings": true,
+        "headers": true,
+        "strict": false,
+        "stern": false
+      },
+    "MD014": false,
+    "MD033": false,
+    "MD040": false
+}
```
Comment (holgerson97) [inline .github/rules/markdownlint.json:17]: Please provide a small documentation why you choosed these settings.

File: .github/workflows/markdownlint.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+name: Markdown Lint
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
```
Comment (holgerson97) [inline .github/workflows/markdownlint.yml:9]: This is a repository for templated workflows. Please make this workflow reusable.

Check: https://docs.github.com/en/actions/using-workflows/reusing-workflows

**Review by holgerson97 — CHANGES_REQUESTED** [review]
Please provide a simple documentation on how to call this workflow when switched to reusable workflows.

File: .github/workflows/markdownlint.yml (hunk 2 above)
Comment (framtale) [reply .github/workflows/markdownlint.yml]: Done. Ready for review

File: .github/rules/markdownlint.json (hunk 1 above)
Comment (framtale) [reply .github/rules/markdownlint.json]: Where should I add the documentation since .json does not support comments?

**Review by holgerson97 — COMMENTED** [review]
LGTM

File: .github/workflows/markdownlint.yml (hunk 2 above)
Comment (holgerson97) [reply .github/workflows/markdownlint.yml]: Create folder docs/ and put Markdown file in there.

File: .github/rules/markdownlint.json (hunk 1 above)
Comment (holgerson97) [reply .github/rules/markdownlint.json]: Create folder docs/ and put Markdown file in there.

Comment (framtale) [reply .github/rules/markdownlint.json]: Done

**Review by holgerson97 — APPROVED** [review]
LGTM

**Pull request merged**